coala_quickstart/coala_quickstart.py

```diff
@@ -1,35 +1,31 @@
-#!/usr/bin/env python3
+from pyprint.ConsolePrinter import ConsolePrinter
```

Comment:
I felt `pyprint` is more of an external library than `coalib` or `coala_quickstart`. So I placed it above the rest - still change?